UnityProject/Assets/Scripts/HealthV2/Living/Damage/BodyPartDamage.cs

```diff
@@ -69,6 +69,13 @@ public partial class BodyPart
 		/// </summary>
 		public DamageSeverity Severity = DamageSeverity.LightModerate;
 
+		/// <summary>
+		/// How much damage can this body part last before it breaks/gibs
+		/// <summary>
+		public float DamageThreshold = 18f;
+		public DamageSeverity GibsOnSeverityLevel = DamageSeverity.Max;
```

Comment:
imo, you should never lose a body part as an it gets destroyed, 
it can be severely damaged, but as long as you pick up all the body parts Heal them up and sew them all back together you can bring someone back

Reply:
oh yeah, later on as well the brain should be invincible as in nothing should be able to destroy it

Reply:
It doesn't get destroyed, it falls off on the ground.

Reply:
maybe rename to dismember then

Reply:
Done 👍